bot/kodiak/test_evaluation.py

```diff
@@ -290,6 +294,57 @@ def test_config_fixtures_equal(config_str: str, config: V1) -> None:
     assert config == V1.parse_toml(config_str)
 
 
+async def mergeable(
```

Comment:
this is the start of making each test case in test_evaluation.py have less noise by providing defaults to override.

Reply:
nice to add a doc string for that